internal/npm_install/npm_install.bzl

```diff
@@ -96,6 +96,22 @@ See https://github.com/bazelbuild/bazel/issues/5153. If
 you are running into performance issues due to a large
 node_modules target it is recommended to switch to using
 fine grained npm dependencies.
+""",
+    ),
+    "manual_package_build_file_contents": attr.string_dict(
```

Comment:
it's awfully subtle to explain to users what's the difference between manual_build_file_contents and manual_package_build_file_contents. It would be much better to fix the one feature to do what you need than to add a second one.